zerver/webhooks/front/tests.py

```diff
@@ -0,0 +1,282 @@
+from typing import Text
+import ujson
+
+from zerver.lib.test_classes import WebhookTestCase
+
+class FrontHookTests(WebhookTestCase):
+    STREAM_NAME = 'front'
+    URL_TEMPLATE = "/api/v1/external/front?&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'front'
+
+    def _test_no_message_data(self, fixture_name: Text) -> None:
+        payload = self.get_body(fixture_name)
+        payload_json = ujson.loads(payload)
+        del payload_json['conversation']['subject']
```

Comment:
And why do we need to delete something from the payload. This is usually a bad thing to do. We shouldn't be touching the payloads. And if there is a payload that doesn't contain the data you are deleting that should be a different payload altogether.